tools/autodeployment/webhook.js

```diff
@@ -67,41 +67,24 @@ function getSha(eventType, githubData) {
 }
 
 /**
- * Create a handler for the particular GitHub push event and build type
- * @param {string} buildType - one of `production` or `staging`
- * @param {string} gitHubEvent - the GitHub Push Event name
+ * @otcokit/webhooks .on
+ * https://github.com/octokit/webhooks.js#webhookson
  */
-function handleEventType(buildType, gitHubEvent) {
-  if (DEPLOY_TYPE !== buildType) {
-    return;
-  }
-
-  /**
-   * @otcokit/webhooks .on
-   * https://github.com/octokit/webhooks.js#webhookson
-   */
-  webhooks.on(gitHubEvent, (event) => {
-    const githubData = event.payload;
-
-    if (
-      requestFilter(
-        githubData.repository.name,
-        buildType,
-        githubData.action,
-        githubData.ref,
-        githubData.repository.master_branch
-      )
-    ) {
-      const sha = getSha(gitHubEvent, githubData);
-      addBuild(buildType, githubData, sha);
-    }
-  });
-}
-
-// Production builds happen when GitHub sends us a `release` event
-handleEventType('production', 'release');
+webhooks.on('push', (event) => {
+  const githubData = event.payload;
 
-// Staging builds happen when GitHub sends us a `push` event
-handleEventType('staging', 'push');
+  if (
+    requestFilter(
```

Comment:
I'm unclear if we will still want to do this or not.

Reply:
I think we need a modified version of this. We need to check if this was a push to `master` or not, otherwise we'll deploy after every push (which we don't want).

Reply:
I wonder if we can be simpler than this: does it have `deploy_tag`?  If it does, deploy it.  If not, ignore.